charts/nginx-gateway-fabric/templates/deployment.yaml

```diff
@@ -103,6 +103,12 @@ spec:
         {{- if .Values.nginxGateway.gwAPIInferenceExtension.enable }}
         - --gateway-api-inference-extension
         {{- end }}
+        {{- if (and .Values.nginxGateway.gwAPIInferenceExtension.enable .Values.nginxGateway.gwAPIInferenceExtension.endpointPicker.disableTLS) }}
+        - --endpoint-picker-disable-tls
+        {{- end }}
+         {{- if .Values.nginxGateway.gwAPIInferenceExtension.enable }}
```

Comment:
Good call on putting both these flags behind the gwAPIInferenceExtension.enable flag!

You only need to specify it once though for both flags, so you can remove the first `{{-end }}` and second `{{- if<...>`